libraries/localization-utilities/src/TypingsGenerator.ts

```diff
@@ -62,7 +89,31 @@ export class TypingsGenerator extends StringValuesTypingsGenerator {
           });
         }
 
-        return { typings };
+        let exportAsDefaultInterfaceName: string | undefined;
+        if (inferDefaultExportInterfaceNameFromFilename) {
+          const lastSlashIndex: number = Math.max(filePath.lastIndexOf('/'), filePath.lastIndexOf('\\'));
+          let extensionIndex: number = filePath.lastIndexOf('.');
+          if (filePath.slice(extensionIndex).toLowerCase() === '.json') {
+            extensionIndex = filePath.lastIndexOf('.', extensionIndex - 1);
```

Comment:
How does this handle scenarios where it just ends with `.json`? Or is that not a scenario that this would ever encounter?

Reply:
That should never happen, since that isn't one of the extensions listed in the config.